docker-compose/README.md

```diff
@@ -0,0 +1,23 @@
+# Docker examples for lamassu
+
+This directory contains docker-compose examples for lamassu.
+
+## Usage
+
+Make sure you have built lamassu (check for jar file in `..target/`).
+
+Edit `application-config/feedproviders.yml` so that it contains at least one source feed.
+
+The run docker-compose:
+
+    docker-compose -f docker-compose-simple.yml up
+
+or
+
+    docker-compose -f docker-compose-advanced.yml up
```

Comment:
@hbruch This example is possible a bit contrived, since there is no scaling going on, and there is also just a single instance of redis, but I left it there to demonstrate how to run it.

Reply:
To avoid performance degradation, would it be sufficient to have a non-leader lamassu instance on the same redis, or would one need to operate a second redis and sync between both instances?

Reply:
You're right there's no need to have a second redis instance just because you split the deployment of lamassu.

Reply:
In fact, you could even run with multiple instances of redis (read replicas), without splitting the lamassu deployment. So not really an issue for this example.

Reply:
Ok. What would you advise for a production scenario with > 20 feeds to ingest? Two lamassu instances and two redis instances, one of them as leader, on as read replica?